lib/networks/client.d.ts

```diff
@@ -24,6 +24,8 @@ export declare abstract class Client {
     approve(tokenAddress: string, spender: string, amount: string): Promise<string>;
     increaseAllowance(tokenAddress: string, spender: string, additionalAmount: string): Promise<string>;
     allowance(tokenAddress: string, spender: string): Promise<bigint>;
+    getDirectDepositContract(poolAddress: string): Promise<string>;
+    directDeposit(poolAddress: string, amount: string, zkAddress: string): Promise<string>;
```

Comment:
I guess it should be ddAddress or something like that.

Reply:
No, here is a Pool contract address passed as a first parameter. DD contract address is fetched under-the-hood

Reply:
`getDirectDepositContract ` method returns ddAddress to approve tokens to the right address in the console

Reply:
Got it, thanks